.github/workflows/WebPageToMarkdown.yml

```
@@ -31,9 +25,19 @@ jobs:
           githubToken: ${{ github.token }}
 
       - name: Git Auto Commit
+        if: ${{ env.issue_body && startsWith(env.issue_title, '[Auto]') }}
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
           commit_message: '${{ github.event.issue.title }}'
           file_pattern: chinese/articles/*.md
           commit_user_name: PageToMarkdown Bot
           commit_user_email: PageToMarkdown-bot@freeCodeCamp.org
+
+      - name: Issue does not need to generate markdown
+        if: ${{ !startsWith(env.issue_title, '[Auto]') }}
+        shell: bash
+        run: echo "This issue does not need to generate a markdown file." 1>&2; exit 1;
```

```suggestion
        run: echo "This issue does not need to generate a markdown file." 1>&2; exit 1;
```
这么简单的语句，所有 shell 都支持。

---

> 这么简单的语句，所有 shell 都支持。

`1 > &2` 不兼容